x/icaauth/module_ibc.go

```diff
@@ -38,13 +40,17 @@ func (am IBCModule) OnChanOpenInit(
 	chanCap *capabilitytypes.Capability,
 	counterparty channeltypes.Counterparty,
 	version string,
-) error {
+) (string, error) {
```

Comment:
is icaauth ok with future channel upgrades?

Reply:
I don't think there are any changes required in `chain-main`. But, I think, we'll need some changes in `ibc-go` ([here](https://github.com/cosmos/ibc-go/blob/9f70a070d773f8bfdb62d4205d8878f3149f351a/modules/apps/27-interchain-accounts/controller/keeper/account.go#L45)). I've asked Carlos about this.